src/main/java/appeng/core/Registration.java

```diff
@@ -19,6 +19,7 @@
 package appeng.core;
 
 
+import appeng.recipes.game.*;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Using the '.*' form of import should be avoided - appeng.recipes.game.*. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck)